File: api/Practica.Common/Models/UsuarioViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Practica.Common.Models
{
    public class UsuarioViewModel
    {
        public int Usua_Id { get; set; }
        public string Usua_Usuario { get; set; }

        public string Usua_Color { get; set; }
        public string Usua_Contra { get; set; }
        public int? Usua_Admin { get; set; }
        public int? Empl_Id { get; set; }
        public int? Rol_Id { get; set; }
        public string Rol_Descripcion { get; set; }
        public string Usuario { get; set; }
        public int? Usua_Usua_Creacion { get; set; }
        public DateTime? Usua_Fecha_Creacion { get; set; }
        public int? Usua_Usua_Modifica { get; set; }
        public DateTime? Usua_Fecha_Modifica { get; set; }

        [NotMapped]
        public string Usua_Creacion { get; set; }

        [NotMapped]
        public string Usua_Modifica { get; set; }
    }
}

File: app/plantilla/src/app/demo/models/UsuarioViewModel.ts
export class Usuario{
    usua_Id!: number;
    usua_Usuario!:string;   
    empleado!: string;
    rol_Descripcion!: string;
    usua_Admin!: number;  
    empl_Id!: number;
    rol_Id!: number;  
    usua_Contra!: string;
    Usua_VerificarCorreo?: string;
}

export class Fill{
    usua_Id!: number;
    usua_Usuario!:string;   
    empleado!: string;
    rol_Descripcion!: string;
    usua_Admin!: number;  
    empl_Id!: number;
    rol_Id!: number;  
    usua_Contra!: string;
    usua_Creacion!:string;   
    usua_Fecha_Creacion!:string;   
    usua_Modifica!:string;   
    usua_Fecha_Modifica!:string;  
    Usua_VerificarCorreo?: string;
}

export class FillPerfilUsuario{
    usua_Id?: number;
    usua_Usuario: string;
    usua_Color: string;
    usuario?: string;
}